docs/tutorials/dsd_2022_demo/data/structures.ini

```diff
@@ -0,0 +1,92 @@
+# written by HYDROLIB-core 0.3.1
+
+[General]
+    fileVersion = 2.00
+    fileType    = structure
+
+[Structure]
+    id                     = 636
+    name                   = De Poale 34 m3/min
+    type                   = pump
+    branchId               = 164
+    chainage               = 610.198737
+    orientation            = positive
+    controlSide            = suctionSide
+    numStages              = 1
+    capacity               = 0.567
+    startLevelSuctionSide  = -0.1
+    stopLevelSuctionSide   = -0.2
+    startLevelDeliverySide = 0.0
+    stopLevelDeliverySide  = 0.0
+    numReductionLevels     = 1
+    head                   = 0.0
+    reductionFactor        = 1.0
+
+[Structure]
+    id                 = 8_1476
+    name               = id         # Given name in the user interface.
+    type               = orifice
+    branchId           = 457
+    chainage           = 211.274378
+    allowedFlowDir     = positive
+    crestLevel         = -1.0
+    crestWidth         = 1.15
+    gateLowerEdgeLevel = 3.0
+    corrCoeff          = 0.63
+    useVelocityHeight  = 1
+    useLimitFlowPos    = 0
+    useLimitFlowNeg    = 0
+
+[Structure]
+    id                     = DV_8_216
+    name                   = id          # Given name in the user interface.
+    type                   = pump
+    branchId               = 459
+    chainage               = 174.381882
+    orientation            = positive
+    controlSide            = suctionSide
+    numStages              = 1
+    capacity               = 0.05
+    startLevelSuctionSide  = 0.4
+    stopLevelSuctionSide   = 0.0
+    startLevelDeliverySide = 0.0
+    stopLevelDeliverySide  = 0.0
+    numReductionLevels     = 1
+    head                   = 0.0
+    reductionFactor        = 1.0
+
+[Structure]
+    id            = 636 [compound]
+    name          = De Poale 34 m3/min
+    type          = compound
+    numStructures = 1
+    structureIds  = 636
+
+[Structure]
```

Comment:
This compound should be removed (the underlying DV-1472 does not exist in start model).